Source/Validation.swift

```diff
@@ -184,6 +191,34 @@ extension DataRequest {
     }
 }
 
+extension DataStreamRequest {
+    public typealias Validation = (_ request: URLRequest?, _ response: HTTPURLResponse) -> ValidationResult
+
+    public func validate<S: Sequence>(statusCode acceptableStatusCodes: S) -> Self where S.Iterator.Element == Int {
```

Comment:
Missing `@discardableResult`.

Reply:
Addressed in https://github.com/Alamofire/Alamofire/pull/3074/commits/52e4f454339fac1e5ad7009e631820f7c9aad107.